order/interface_test.go

```diff
@@ -190,10 +190,22 @@ func TestOrderReservedValue(t *testing.T) {
 			}
 
 			// For bids the taker pays the most fees if the min
-			// units get matched every block.
+			// units get matched every block. There's an edge case
```

Comment:
Looks like we have test cases for the remainder case (bid and ask of size 10 with min size 4). Could also add a case for the non-remainder case for a `MinUnitsMatch` that is not the base unit?